build-package.sh

```diff
@@ -532,6 +537,9 @@ while (($# >= 1)); do
 				termux_error_exit "./build-package.sh: option '--library' requires an argument"
 			fi
 			;;
+		--disable-dependency-download-parallelizing)
```

Comment:
It might also be nice to be able to specify a maximum amount of parallel downloads.
e.g. `--parallel 8`.

But since that's out of scope for this PR I'll just leave it as a potential improvement for later.
That option name is still way too long though.
```suggestion
		--no-parallel)
```